CONTRIBUTING.md

```diff
@@ -70,6 +70,25 @@ contents of the `web-design-standards` project under `node_modules/uswds` within
 the `project-using-npm`. For more information on `npm-link`, you can read about
 it in the [`npm-link` documentation](https://docs.npmjs.com/cli/link).
 
+## Local Gem Development for the Jekyll Documentation Website
```

Comment:
I think simplifying might help, something like "Local Development with Jekyll Documentation Website". I also question whether "Documentation Website" is the right term. Would contributors know what this is? Maybe we could show the link like "Documentation Website (playbook.cio.gov/designstandards)